packages/neon_framework/lib/src/storage/persistence.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart' show protected;
+
+/// A persistent key value storage.
+abstract interface class Persistence<T extends Object> {
+  /// Whether a value exists at the given [key].
+  FutureOr<bool> containsKey(String key);
+
+  /// Completes with true once all values have been cleared.
```

Comment:
```suggestion
  /// Clears all values.
```

Reply:
This does not explain what the returned value does.

Reply:
But this is also not explained for all the other methods.